### ISSUE TITLE
[WEBUI] Editing udf.rs should not restart the SQL compiler

### ISSUE THREAD

**Issue by mihaibudiu** [feldera/feldera#2905]
Today it does.

**Comment by mihaibudiu** [comment]
Is this done?

**Comment by snkas** [comment]
No, though SQL compilation is no longer queued behind Rust compilation. The gain for this as SQL compilation is pretty fast is not that much anymore. It requires some additional logic. Keep open for now to keep track of this as a low priority todo.
